.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+<!--- All code change PRs should relate to an issue, reference it here; see example below -->
+<!--- Closes: #123 -->
+
+## Description
+<!--- Describe your changes in detail; the what, the why, any findings, etc -->
+<!--- Include any screenshots of changed UI; Include any gifs if it was a flow / UX change -->
+
+## How Has This Been Tested?
+<!--- Please describe in detail how you tested your changes. -->
+<!--- Include details of your testing environment, and the tests you ran to -->
+<!--- see how your change affects other areas of the code, etc. -->
+
+## Request review criteria:
+<!--- This PR will be merged by any repository approver when it meets all the points in the checklist -->
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+
+- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
+- [ ] Included any necessary screenshots or gifs if it was a UI change.
+- [ ] Included tags to the UX team if it was a UI/UX change.
+- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
+- [ ] The developer has manually tested the changes and verified that the changes work
```

Comment:
As we discussed, it would be nice to have an entry to ensure testing coverage, and now that we've integrated the bump into storybook integration, I think it should be fine to include it in this PR